Source/Project/Sizer/Sizer.f90

```diff
@@ -301,6 +322,7 @@ FUNCTION sizeFunctionMinimumOnBox( sizer, xMin, xMax ) RESULT(hMin)
          REAL(KIND=RP)                    :: cSize, aSize
          REAL(KIND=RP)                    :: cHeight, cWidth, cDim
          INTEGER                          :: i,j, nX(3) = [10,10,0]
+         REAL(KIND=RP), PARAMETER         :: SIZE_FACTOR = 0.9_RP
```

Comment:
Is this factor set such that the size does not "accidentally" make `hMin` smaller than the background grid requested in the control file?

Reply:
This is in there to bail early out of the loop because gaussianCurvatureAt is so expensive.

Reply:
This is in there to bail early out of the loop because gaussianCurvatureAt is so expensive.

Reply:
Okay, that makes sense.